index.ts

```diff
@@ -113,32 +142,33 @@ function candidateToGeojson (candidate) {
  * @param {string} [$0.url]                       optional URL to override ESRI suggest endpoint
  * @return {Promise}                              A Promise that'll get resolved with the suggest result
  */
-export function autocomplete ({
+export function autocomplete({
+  boundary,
   clientId,
   clientSecret,
-  boundary,
   focusPoint,
   text,
   url
 }: BaseQuery & {
-  boundary?: Boundary,
-  focusPoint?: any,
-  text: string,
-}): Promise<BaseResponse> {
-  const geocoder = getGeocoder(clientId, clientSecret, url)
-  const options = {}
+  boundary?: Boundary
+  focusPoint?: Point
+  text?: string
+}): Promise<Array<SuggestOption> & { query: { text: string } }> {
```

Comment:
This is missing the `features` key.

```suggestion
}): Promise<{
  features: Array<SuggestOption>
  query: { text: string } 
}> {
```

Reply:
Have fixed all endpoints in 084c5d3b0eaa3897e885428831db509ede8e5391